src/v2/manifest/manifest_schema2.rs

```diff
@@ -61,16 +79,82 @@ pub struct Platform {
     pub features: Option<Vec<String>>,
 }
 
+impl ManifestSchema2Spec {
+    /// Get digest of the configuration object referenced by this manifest.
```

Comment:
This does not return the digest, though. In which direction do you want to fix it?